TGM/src/main/java/network/warzone/tgm/match/MatchManifest.java

```diff
@@ -81,4 +83,14 @@
         modules.add(new MapCommandsModule());
         return modules;
     }
+
+    /**
+     * Modules that are added depending on configuration
+     * @return
+     */
+    public List<MatchModule> allocateConditionalModules() {
```

Comment:
It'd probably be best if the match summary was an option for map makers rather than a config option

Reply:
perhaps